apps/storybook-react-native/package.json

```diff
@@ -1,8 +1,8 @@
 {
   "name": "@metamask/storybook-react-native",
-  "version": "0.0.0",
   "private": true,
   "description": "Storybook React Native",
+  "files": [],
```

Comment:
Part of the yarn constraints that we do want private non publishable packages to adhere to https://github.com/MetaMask/metamask-design-system/blob/main/yarn.config.cjs#L172-L176